ruby_exercises/mythical-creatures/lib/dragon.rb

```diff
@@ -0,0 +1,31 @@
+class Dragon
+  attr_reader :name,
+              :color
+
+  attr_accessor :rider,
+                :hungry
+
+  def initialize(name, color, rider)
+    @name = name
+    @color = color 
+    @rider = rider
+    @hungry = true
+  end 
+
+  def hungry?
+    @hungry
+  end
+
+  def eat 
+    count = 0
+
+    while count < 4 do
```

Comment:
So instead of putting this into a while loop, you can use the count to just do a conditional check.

```
count = 0

if count <4
   count +=1
else (or else if count >= 4)
   @hungry = false
end
```